storage/fileservice/clean_test.go

```diff
@@ -1 +1,46 @@
 package fileservice
+
+import (
+	"os"
+	"testing"
+
+	"github.com/kuoss/lethe/config"
+	"github.com/kuoss/lethe/util/testutil"
+	"github.com/stretchr/testify/require"
+	// "github.com/stretchr/testify/require"
```

Comment:
```suggestion
	"github.com/stretchr/testify/require"
```
완전하게 동일한 import문이 주석 처리 되어 있어서 삭제하셔도 좋을 것 같습니다.

Reply:
주석 삭제했습니다.